tests/extractors/test_bigquery_extractor.py

```diff
@@ -0,0 +1,44 @@
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import unittest
+import logging
+
+from marquez_airflow.extractors.bigquery_extractor import BigQueryExtractor
+
+
+class TestBigQueryExtractor(unittest.TestCase):
+    def setUp(self):
+        logging.basicConfig(format='%(levelname)s:%(message)s',
+                            level=logging.DEBUG)
+        logging.debug("TestBigQueryExtractor.setup(): ")
+
+    def test_extract(self):
+        logging.info("test_extractor")
+
+        task = Task(dag_id="dag_id", task_id="task_id",
+                    sql="select name, city from customers;")
+
+        step_meta = BigQueryExtractor(operator=None).extract(task)
+
+        assert 'customers' == step_meta.outputs[0]
+
+
+class Task:
```

Comment:
We'd want to use the BigQuery operator to instantiate a task in the tests and not define our own.

Reply:
corrected.